src/nanoFramework.IoT.Device.CodeConverter/appsettings.json

```diff
@@ -1,10 +1,10 @@
 {
-  "SourceDirectory": "e:\\GitHub\\DotNet-iot\\src\\devices",
+  "SourceDirectory": "c:\\Repos\\iot\\src\\devices",
```

Comment:
Everyone that touches this seems to be fighting to put it's local path! 🤣

Reply:
:-D Does it really matter? :-)

Reply:
No. Was just noticing that and found it amusing. 😉